src/api/article/article.model.js

```diff
@@ -46,119 +44,88 @@ const joiArticleSchema = Joi.object({
   // Images are as described in 'covers' guide by Vitalik.
   images: Joi.object().required(),
   // Can only be present when Article type is video.
-  video: joiVideoSchema,
-  color: Joi.color().default('#000000'),
+  video: Joi.object({
+    platform: Joi.string().valid(['youtube']),
```

Comment:
shouldn't platforms (youtube or soundcloud) be extracted as constants?

Reply:
I think it's redundant for now